neural_compressor/conf/pythonic_config.py

```diff
@@ -18,7 +18,7 @@
 import logging
 from .dotdict import DotDict
 from ..config import _BaseQuantizationConfig, accuracy_criterion, BenchmarkConfig, \
-                     check_value, DistillationConfig, options, PruningConfig
+                     check_value, DistillationConfig, options, WeightPruningConfig
```

Comment:
here should we introduce new WeightPruningConfig and keep old PruningConfig as the latter is used by 1.x pruning implementation?

Reply:
so pls check if INC 1.14.2 includes PruningConfig or not. if yes, it means 1.x pruning code used this definition before and should keep and add new WeightPruningConfig but not replace?

Reply:
As you mentioned before, PruningConfig should be only used in 2.0a. I have checked  neural_compressor/config.py and nerual_compressor/conf/config in v1.4.2, this class has not been defined

Reply:
> here should we introduce new WeightPruningConfig and keep old PruningConfig as the latter is used by 1.x pruning implementation?
As you mentioned before, PruningConfig should be only used in 2.0a. I have checked neural_compressor/config.py and nerual_compressor/conf/config in v1.4.2, this class has not been defined